test_naucse/test_views.py

```diff
@@ -3,14 +3,15 @@
 
 from naucse.views import make_app
 
-from test_naucse.conftest import fixture_path
+from test_naucse.conftest import fixture_path, setup_repo
 
 @pytest.fixture
 def client(monkeypatch):
-    monkeypatch.setenv('NAUCSE_ROOT_PATH', fixture_path / 'test_content')
+    monkeypatch.setenv('NAUCSE_ROOT_PATH', str(fixture_path / 'test_content'))
 
     app = make_app()
     app.config['DEBUG'] = True
+    app.config['TRAP_HTTP_EXCEPTIONS'] = True
```

Comment:
TRAP_HTTP_EXCEPTIONS is set in make_app **and** here. I suppose it should only be set when testing... ?

Reply:
It should definitely be set when testing.
But it should also be set when freezing, so you get a good traceback, rather than an error page and a “bad status” traceback from frozen-flask.
If naucse ever becomes a dynamic app, the line here should be removed but the one in tests should stay.

I'll add comments.